src/charactersheet/services/character/status_service_components/healthiness_component.js

```diff
@@ -53,13 +53,17 @@ export function HealthinessStatusServiceComponent() {
 
             if (deathSaves.length > 0) {
                 for(var i=0; i<3; i++) {
-                    if(deathSaves[i].deathSaveSuccess()) {
-                        deathSaveSuccesses++;
+                    if (deathSaves[i] !== undefined) {
```

Comment:
I think this can all be one line, like:
```
if (deathSaves[i] !== undefined && deathSaves[i].deathSaveSuccess()) {
  deathSaveSuccesses++;
}
```

Please update both of these statements.